src/query/storages/fuse/fuse/src/operations/gc.rs

```diff
@@ -131,6 +138,29 @@ impl FuseTable {
             }
         }
 
+        // 3.3 Find all the table statistic files need to be deleted
+        {
+            if let Some(root_ts_location) = root_ts_location_opt {
+                let start = Instant::now();
+                let snapshots_io = SnapshotsIO::create(
+                    ctx.clone(),
+                    self.operator.clone(),
+                    self.snapshot_format_version().await?,
+                );
+                let ts_to_be_purged = snapshots_io
+                    .read_table_statistic_files(&root_ts_location, None)
```

Comment:
will the `root_ts` be included in the `ts_to_be_purged`?

Reply:
no, exclude the `root_ts` file.